acra/keys/inventory.md

```diff
@@ -41,60 +45,53 @@ Storage keys can be represented by either:
-    and are decrypted by AcraServer upon users’ login.
+- **AcraServer** should have:
+    - AcraConnector's transport public key and AcraServer's own transport private key. This is necessary for accepting connections from clients via Secure Session
+    - Storage and zone asymmetric private key(s) for decrypting AcraStructs generated by AcraWriter.
+    - If AcraServer is running in [Transparent proxy mode]({{< ref "/acra/configuring-maintaining/general-configuration/acra_server.md#transparent-proxy-mode-INVALID" >}}), it should possess the zone and storage public key(s) for encrypting the values from SQL queries to AcraStructs and symmetric data encryption keys for AcraBlocks.
+    - Poison record keys to be able to detect poison records in database responses.
+    - Keys for secure logging that prevents tampering messages, removing, adding or changing the order of log entries
+    - Authentication storage key to access AcraWebConfig's credentials and allow update AcraServer's configuration from WebUI.
 
-**AcraTranslator** needs to have the following keys:
+- **AcraTranslator** needs to have same keys as AcraServer except authentication storage key.
 
-  - AcraTranslator’s own transport private key along with AcraConnector’s transport public key.
+- **AcraWriter** should have the storage public key(s). They are necessary for encrypting data into AcraStructs in such a way that would only be readable for AcraServer/AcraTranslator.
 
-    They are necessary for accepting connections from the clients via Secure Session.
+- If you're using **AcraWebConfig** HTTP server to configure AcraServer remotely, the users' credentials are stored encrypted with authentication storage key and are decrypted on AcraServer upon users' login. AcraServer needs to have an authentication storage key.
 
-  - Storage private keys for decrypting AcraStructs generated by AcraWriter.
+#### Keys table
 
-**AcraWriter** should have the storage public keys.
+##### Asymmetric keys
+| Purpose  | Private key  | Stays on  | Public key | Put to
+| --- | --- | --- | --- | ---
+| Transport AcraConnector | clientid | AcraConnector | clientid.pub | AcraServer<br/>AcraTranslator
+| Transport AcraServer | clientid_server| AcraServer | clientid_server.pub | AcraConnector
+| Transport AcraTranslator | clientid_translator| AcraTranslator | clientid_translator.pub | AcraConnector
+| AcraStruct encryption |  |  | clientid_storage.pub | AcraWriter<br/>AcraTranslator<br/>AcraServer (Transparent proxy mode)
+| AcraStruct decryption | clientid_storage | AcraServer<br/>AcraTranslator |  |  |
+| AcraStruct encryption with Zones |  |  | zoneid_zone.pub | AcraWriter<br/>AcraTranslator<br/>AcraServer (Transparent proxy mode)
+| AcraStruct decryption with Zones | zoneid_zone | AcraServer<br/>AcraTranslator | |
+| Poison record with AcraStruct | poison_key | AcraServer<br/>AcrTranslator | poison_key.pub | AcraServer<br/>AcrTranslator
```

Comment:
I think it would be nice to wrap key file names into backticks rendering them `monospace` to signify that these are file names.

Another thing that I would do is making it obvious that `clientid` is not use verbatim but rather a placeholder. Given that there are examples with specific client IDs below, I think it would be okay to use `${clientID}` or something like this to denote substitution. Hopefully, this would be enough of a hint.

Ditto for symmetric keys table below.